content/tutorials/marketplace-fees.md

```diff
@@ -0,0 +1,84 @@
+
+The following fees apply to the consumption of data assets as a part of Consume fee:
+
+- Consume Market Consumption Fee
+- Publisher Market Consumption Fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently. This fee is set to 0 (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.1% is applicable on all swaps using another token as the base token.
+
+### Provider fee
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starting compute jobs, encryption of DDOs, and decryption of DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click here.
+
+Provider fees are paid to the individual or organization running their Provider instance whenever a user consumes a data asset or swaps a datatoken for a basetoken. These fees can be set to an absolute amount, and the receiver can also decide which token to receive the fees in - they don't have to be in the same currency used in the consuming market.
+
+Provider fee can also be used to charge for computing resources. Based on the compute resources needed to run an algorithm in the Compute-to-Data environment, a consumer can choose the amount of pay according to their needs.
+
+This fee incentivizes individuals and organizations to run their provider instances and charge consumers according to resource usage.
+
+## Fee values
+
+| Swap Fees (Fixed Price or Auto)                                                                                       | Value in Ocean Market, using any Provider                                                                                                                                                                                 | Value in Other Markets                                       |
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| publishMarket: Pools                                                                                                  | Set in market config, by OPF Value = 0 (agreed Feb 28 by T, A, Mihai, Matthias, Ana)                                                                                                                                      | Set in market config, by publishMarket                       |
```

Comment:
@akshay-ap Please remove all the references of "agreed on" and the members. This was relevant for us internally